app/views/upload.scala.html

```diff
@@ -48,4 +49,13 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
         </form>
     </div>
 </div>
+<div id="progress-bar" class="govuk-grid-row hide">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">@Messages("fileProgress.header")</h1>
+        <p class="govuk-body">@Messages("fileProgress.title")</p>
+        <div>
+            <progress class="progress-display" value="" max="50"></progress>
```

Comment:
I've been thinking about the classes and IDs and BEM. I think this element should be "progress-bar" rather than "progress-display" because "progress-display" sounds like it might include other elements, such as text that describes the progress.

Reply:
I had initially named it progress-bar but Tom mentioned that it can be confusing having an id and class with the same value, so I changed it to progress-display, could there possibly be a better word besides display that I could use here?